src/font-style.css

```diff
@@ -1,23 +1,22 @@
 /* FONT-FAMILY  */
-
 @font-face {
   font-family: ITC-Stone-Serif-Small-Caps-Bold;
-  src: url('./components/fonts/ITC-Stone-Serif-Small-Caps-Bold.ttf');
+  src: local('./components/fonts/ITC-Stone-Serif-Small-Caps-Bold.ttf');
```

Comment:
This trick saves time and resources for those users who already have the font installed. 💯 
However, if the user doesn't have it, the browser will just use a sans-serif font. 😢 
I looked it up and found a StackOverflow [article](https://stackoverflow.com/questions/3837249/font-face-src-local-how-to-use-the-local-font-if-the-user-already-has-it) with a simple solution for this situation.

Reply:
Thanks for the link! I solved it!